components/RealTimeOrderUpdate.js

```diff
@@ -19,19 +19,26 @@ class RealTimeOrderUpdateBase extends Component {
       meta,
       topics,
       handleOnMessage,
+      handleOnConnect,
       id,
     } = this.props;
 
+
+
     return (
       <SockJsClient url={url} topics={[topics]}
         onMessage={(data) => {
           handleOnMessage(data, id)
           console.log(`get device: ${Device.deviceName}-${Device.modelName}`)
+
         }}
         ref={(client) => {
           this.orderFormRef = client
         }}
         onConnect={() => {
+          if (this.props?.webSocketUrlPath) {
```

Comment:
On another thought, perhaps we can simplify this logic to just call handleOnConnection(this.orderFormRef) and remove the if check. Can discuss.

Reason:
> other usage without on connect specific logic will just ignore this.
> all is required from RealTimeOrderUpdate is this.orderFormRef. Other parameters can be obtained directly.